physics/clm_lake.f90

```diff
@@ -238,7 +238,7 @@ subroutine calculate_z_dz_lake(i,input_lakedepth,clm_lakedepth,z_lake,dz_lake)
       real(kind_lake) :: dz_lake(nlevlake) ! layer thickness for lake (m)
       real(kind_lake) :: depthratio
 
-      if (input_lakedepth(i) == spval) then
+      if (input_lakedepth(i) == spval .or. input_lakedepth(i) < 0.1) then
```

Comment:
Can you add a comment for this new parameter? (minimum lake depth in meters?)

Reply:
This is a workaround for a disturbing bug we have been unable to track down in the RRFS parallel. The lakedepth doesn't match the lake mask. I suspect something went wrong in the cycling, or the orography data was changed during the parallel.

Reply:
```f90
      if (input_lakedepth(i) == spval .or. input_lakedepth(i) < 0.1) then
        ! This is a safeguard against:                                                                                                                                                                                                       
        ! 1. missing in the lakedepth database (== spval)                                                                                                                                                                                    
        ! 2. errors in model cycling or unexpected changes in the orography database (< 0.1)                                                                                                                                                 
```

Reply:
Thanks Sam!